charts/apiclarity/templates/serviceaccount.yaml

```diff
@@ -1,7 +1,9 @@
+{{- if .Values.apiclarity.serviceAccount.create -}}
```

Comment:
Not sure why it is needed, but if we want to support it, should it be on all SA created in all templates?
Also, what about the `serviceAccount` field in the deployments.

Reply:
Good point, I'll make it an option in all templates. Ideally I think I'd like to add an option to override the SA name if this is set to false. It's not necessary to my immediate use-case but it might be a good option to add if you agree...

Reply:
OK, an option to override the SA sounds good. Just make sure all service accounts are covered. Thanks!